src/types/flow.d.ts

```diff
@@ -11,26 +11,16 @@ import { Identifiable, Resource, ResourcePage, Attributes } from './core';
  * Core Flow Base Interface
  */
 export interface FlowBase {
-  slug:
-    | 'addresses'
-    | 'products'
-    | 'brands'
-    | 'collections'
-    | 'categories'
-    | 'customers'
-    | 'cart_items'
-    | 'orders'
-    | 'order_items'
-    | 'promotions'
+  slug: string
```

Comment:
hey @yasiloghmani for context, could you tell us why we need to use the type `string` now?